index.js

```diff
@@ -32,6 +34,7 @@ app.use((req, res, next) => addGenericLocals(req, res, next));
 
 app.use('/cookies', (req, res, next) => {
   res.locals = Object.assign({}, res.locals, req.translate('cookies'));
+  res.locals['session-cookies-table'] = sessionCookiesTable['session-cookies-table'];
```

Comment:
👍 this is the thing that @cmedlandho and @MaxOSullivan might have needed for modern-slavery but we missed
Great spot @SarahJaneLuff